server/data/data-jmap-cassandra/src/test/java/org/apache/james/jmap/cassandra/access/CassandraAccessTokenRepositoryTest.java

```diff
@@ -20,22 +20,35 @@
 package org.apache.james.jmap.cassandra.access;
 
 import org.apache.james.backends.cassandra.CassandraCluster;
-import org.apache.james.jmap.api.access.AbstractAccessTokenRepositoryTest;
-import org.apache.james.jmap.api.access.AccessTokenRepository;
-import org.junit.After;
+import org.apache.james.jmap.api.access.AccessTokenRepositoryTest;
+import org.junit.runner.RunWith;
+import org.xenei.junit.contract.Contract;
+import org.xenei.junit.contract.ContractImpl;
+import org.xenei.junit.contract.ContractSuite;
+import org.xenei.junit.contract.IProducer;
 
-public class CassandraAccessTokenRepositoryTest extends AbstractAccessTokenRepositoryTest {
+@RunWith(ContractSuite.class)
+@ContractImpl(CassandraAccessTokenRepository.class)
+public class CassandraAccessTokenRepositoryTest {
 
-    private CassandraCluster cassandra;
+    private IProducer<CassandraAccessTokenRepository> producer = new IProducer<CassandraAccessTokenRepository>() {
```

Comment:
If I put AccessTokenRepository either on the right or the left of the "=" I 've got an Incompatible type compilation error in IntelliJ.

So I'll keep it like this....